CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 ### Fixed
 
 * Potential NPE in test-harness-core ([#671](https://github.com/spotbugs/spotbugs/issues/671))
+* Support project path with spaces ([#683](https://github.com/spotbugs/spotbugs/issues/683))
```

Comment:
Good. Could you add `in test-harness-core`, then reader can easily understand this fix is for that module.